BSB_LAN/BSB_LAN.ino

```diff
@@ -3839,11 +3855,11 @@ char *build_pvalstr(bool extended) {
   int len = 0;
   outBuf[len] = 0;
   if (extended && decodedTelegram.error != 257) {
-#if !(defined ESP32)
-    len+=sprintf_P(outBuf, PSTR("%4.1f "), decodedTelegram.prognr);
-#else
-    len+=sprintf_P(outBuf, PSTR("%4.1f "), decodedTelegram.prognr);
-#endif
+    if(roundf(decodedTelegram.prognr * 10) != roundf(decodedTelegram.prognr) * 10)
+      len+=sprintf_P(outBuf, PSTR("%4.1f "), decodedTelegram.prognr);
```

Comment:
Not sure right now: Does %4 mean that program numbers are all 4 digits before the dot? Because we have everything from 1 to 5 digits?

Reply:
"4" can be deleted because has no effect on the output

Reply:
Thanks!